semantic_release/__init__.py

```diff
@@ -1,11 +1,13 @@
+"""Semantic Release
```

Comment:
Why was these added?

Reply:
I used pylint to help me catch all of the missing type docs and missing module docstrings showed up as well. I just threw those in there to silence those